packages/api/src/command/job/patient/finish.ts

```diff
@@ -0,0 +1,44 @@
+import { out } from "@metriport/core/util/log";
+import { PatientJob, validateNewJobStatus } from "@metriport/shared";
+import { buildDayjs } from "@metriport/shared/common/date";
+import { FinishJobParams } from "../shared";
+import { getPatientJobModelOrFail } from "./get";
+
+/**
+ * Finishes a patient job.
+ *
+ * @param jobId - The job ID.
+ * @param cxId - The customer ID.
+ * @param forceStatusUpdate - Whether to force the status update (only to be used by internal
+ *                            flows/endpoints).
+ * @param onCompleted - The callback to call once the job is updated.
+ * @returns the updated job.
+ * @throws BadRequestError if the completed status is not valid based on the current state.
+ * @throws NotFoundError if the job doesn't exist.
+ */
+export async function finishPatientJob({
+  jobId,
+  cxId,
+  forceStatusUpdate = false,
+  onCompleted,
+}: FinishJobParams): Promise<PatientJob> {
```

Comment:
Shouldn't we get the end status as a parameter? "finish" is generic in the sense it doesn't carry the meaning of which status to finish the job with. Could be thought as getting it to a final state/status.

NABD to leave as is and update later, though.